packages/squiggle-lang/src/rescript/Reducer/Reducer_Peggy/Reducer_Peggy_Parse.res

```diff
@@ -5,11 +5,15 @@ type node = {"type": string}
 
 @module("./Reducer_Peggy_GeneratedParser.js") external parse__: string => node = "parse"
 
+let syntaxErrorToLocation: Js.Exn.t => Reducer_ErrorValue.location = error => %raw(`error.location`)
```

Comment:
There's probably a more elegant way to do this and I just don't know ReScript well enough. But `Js.Ext` is not documented well and I gave up after ~1h of googling and trying to cast `Js.Ext.t` to peggy's SyntaxError with `genType`...